go/common/crypto/address/address.go

```diff
@@ -0,0 +1,80 @@
+// Package address implements a generic cryptographic address.
+package address
+
+import (
+	"bytes"
+	"encoding/base64"
```

Comment:
Thought we weren't using base64 for these.

Reply:
That was just temporary, I replaced it with Bech32.